python/cugraph/sampling/random_walks.py

```diff
@@ -14,16 +14,12 @@
 import cudf
 from cugraph.sampling import random_walks_wrapper
 import cugraph
-from collections import defaultdict
 
-# FIXME might be more efficient to return either (df + offset) or 3 cudf.Series
 
-
-def random_walks(
-    G,
-    start_vertices,
-    max_depth=None
-):
+def random_walks(G,
+                 start_vertices,
+                 max_depth=None,
+                 use_padding=False):
     """
     compute random walks for each nodes in 'start_vertices'
```

Comment:
outdated description. `use_padding` is not included in the description

Reply:
updated